history/5279.md

```diff
@@ -0,0 +1 @@
+* @barnson: Enable the WiX native SDK when the new Visual C++ Build Tools bundle is installed. Fixes wixtoolset/issues#5279.
```

Comment:
Don't we want the auto association? Add `wixtoolset/issues#5279` in commit comment or PR comment instead?